### PR TITLE
 Added new functions get/setObjectProperty

### PR DESCRIPTION
**Pull request by lex128**
Patch for https://bugs.mtasa.com/view.php?id=9740

### CODE REVIEW

**Comment by CrosRoad95** [comment]
do this allow to enable physics for any object?

**Comment by Einheit-101** [comment]
For me this looks like its just possible to edit some more physics properties of objects, but it doesnt magically convert a static object into a dynamic one.

**Comment by lex128** [comment]
May be better a common function like vehicle handling
```lua
bool setObjectPhysics ( element theObject, string property, var value )
```
and
```lua
table getObjectPhysics ( element theObject )
```

**Comment by qaisjp** [comment]
Your suggestion is fantastic Lex. 

How about `prop getObjectProperty ( element theObject, string prop )`. And a `prop` of `"all"` would return the table of all props?

File: Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp [hunk 1]
```diff
@@ -3951,6 +3980,84 @@ bool CStaticFunctionDefinitions::SetObjectMass ( CClientEntity& Entity, float fM
...
+            Object.SetTurnMass ( fTurnMass );
+            return true;
+        }
+    }
+    return false;
+}
+
+bool CStaticFunctionDefinitions::SetObjectAirResistance ( CClientEntity& Entity, float fAirResistance )
+{
+    //if ( fAirResistance > 0.0f && fAirResistance <= 1.0f )
```
Comment (Citizen01) [inline Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp]: Do you plan to uncomment that out before pushing ? (same for elasticity ?)

Comment (lex128) [reply Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp]: I dont know yet how better

**Comment by lex128** [comment]
Done
```lua
bool  setObjectProperty(element theObject, string property, ...)
...   getObjectProperty(element theObject, string property)
table getObjectProperties(element theObject)
```
And how better `"property"` takes values?
as now like enum
`"mass", "turn_mass", "air_resistance", "elasticity", "center_of_mass", "buoyancy"`

or like vehicle handling
`"mass", "turnMass", "airResistance", "elasticity", "centerOfMass", "buoyancy"`

**Comment by qaisjp** [comment]
Like enum 👍

**Comment by patrikjuvonen** [comment]
Are these related?
- https://bugs.mtasa.com/view.php?id=8777
- https://bugs.mtasa.com/view.php?id=7426

**Comment by qaisjp** [comment]
sorry for the confusion, that last one should be the right one!

**Comment by lex128** [comment]
It's not related @patrikjuvonen

**Pull request merged**